### ISSUE TITLE
Admin SDK call urls wrong

### ISSUE THREAD

**Issue by lobinhojr** [NikiforovAll/keycloak-authorization-services-dotnet#161]
I have the setup below and I am using keycloak 26

//My settings
"Keycloak": {
    "realm": "myrealm-dev",
    "auth-server-url": "http://localhost:8080",
    "ssl-required": "none",
    "verify-token-audience": false,
    "resource": "myrealm-dev-api",
    "credentials": {
      "secret": "<secret>"
    },
    "confidential-port": 0
  }

//On setup
 var options = configuration.GetKeycloakOptions<KeycloakAdminClientOptions>()!;
        
        services.AddDistributedMemoryCache();
        services
            .AddClientCredentialsTokenManagement()
            .AddClient(
                "Keycloak",
                client =>
                {
                    client.ClientId = options.Resource;
                    client.ClientSecret = options.Credentials.Secret;
                    client.TokenEndpoint = options.KeycloakTokenEndpoint;
                }
            );
        
        services
           // TODO: hardcode settings used for force and try urls here on error analysis
          .AddKeycloakAdminHttpClient(new KeycloakAdminClientOptions 
            {
                AuthServerUrl = "http://localhost:8080",
                Realm = "myrealm-dev",
                Resource = "myrealm-dev-api",
            })
            .AddClientCredentialsTokenHandler("Keycloak");

However, when I inject the IKeycloakUserClient on my class and call the method GetUser("myrealm-dev")
var users = await _keycloakUserClient.GetUsersWithResponseAsync("myrealm-dev");

The method return a response with 404 Status.
Analysing the problems and RequestMessage, I realized that url called is wrong
URL called => 
http://localhost:8080/realms/myrealm-dev/admin/realms/myrealm-dev/users

The correct endpoint should be "http://localhost:8080/admin/realms/myrealm-dev/users"

**Comment by NikiforovAll** [comment]
What version do you use?

**Comment by lobinhojr** [comment]
version 2.5.5

**Comment by anperim** [comment]
Same problem. I was using version 2.5.3 and the problem is that /auth (old version of keycloak) was removed and doing the update to 2.5.5 and the problem is exactly as described above.

**Comment by NikiforovAll** [comment]
Yes, sorry, I've unlisted `2.5.5` for this very particular reason, please use `2.6.0`
